telegram_export/__main__.py

```diff
@@ -271,13 +271,28 @@ def find_dialog(dialogs, query, top=25, threshold=0.7):
         config['Dumper']['OutputDirectory'],
         config['TelegramAPI']['SessionName']
     )
+    """
     client = await (TelegramClient(
         absolute_session_name,
         config['TelegramAPI']['ApiId'],
         config['TelegramAPI']['ApiHash'],
         loop=loop,
         proxy=proxy
     ).start(config['TelegramAPI']['PhoneNumber']))
+    """
```

Comment:
Leaving unused code like that isn't generally a good idea. Just remove it. If we need it, we can `git revert`.